File: app/src/main/java/ir/ahfz/rentcar/io/network/model/ClassResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;

public class ClassResponse extends BaseResponse {


    private List<Type> types = null;

    public List<Type> getTypes() {
        return types;
    }

    public void setTypes(List<Type> types) {
        this.types = types;
    }

    public class Type {

        private int id;
        private String _class;
        private String createdAt;
        private String updatedAt;

        public int getId() {
            return id;
        }

        public void setId(int id) {
            this.id = id;
        }

        public String getClass_() {
            return _class;
        }

        public void setClass_(String _class) {
            this._class = _class;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

    }
}

File: app/src/main/java/ir/ahfz/rentcar/io/network/model/ClassResponse.kt
package ir.ahfz.rentcar.io.network.model

import ir.ahfz.rentcar.io.network.BaseResponse
/**
 * Data model for class response
 */
data class ClassResponse(
    var types: List<Type>? = null
) : BaseResponse() {
    /**
     * Data model for Type response
     */
    data class Type(
        var id: Int = 0,
        var class_: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null
    )
}
